.database_generator/mangos_translation/main.py

```
@@ -61,7 +67,7 @@ def import_sql_file(conn, file_path):
     exit(1)
 
 
-def process(locales, version):
+def process(locales, version, datasources={"emit": "inserts", "lightshope": False}):
```

Ruff (0.14.6)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)